website/versioned_docs/version-0.67/updating-upgrade-helper.md

```diff
@@ -0,0 +1,43 @@
+---
+id: updating-upgrade-helper
+title: How to Update Upgrade Helper
+---
+
+_Point people: [@lucasbento](https://github.com/lucasbento), [@pvinis](https://github.com/pvinis)_, [@kelset](https://github.com/kelset)\_
+
+From the readme of [its dedicated repo](https://github.com/react-native-community/upgrade-helper#-how-it-works):
+
+> The Upgrade Helper tool aims to provide the full set of changes happening between any two versions, based on the previous work done in the rn-diff-purge project:
+
+> > This repository exposes an untouched React Native app generated with the CLI react-native init RnDiffApp. Each new React Native release causes a new project to be created, removing the old one, and getting a diff between them. This way, the diff is always clean, always in sync with the changes of the init template.
+
+The upgrade helper webapp relies on [rn-diff-purge](https://github.com/react-native-community/rn-diff-purge) having the diff for the versions of RN released.
+
+## Instructions to update
+
+### For release versions >= 0.68
+
+- The Github action will be triggered by the [pCircleCI job that pushes the release link](https://github.com/facebook/react-native/blob/main/.circleci/config.yml#L822)
```

Comment:
```suggestion
- The GitHub action will be triggered by the [CircleCI job that pushes the release link](https://github.com/facebook/react-native/blob/main/.circleci/config.yml#L822).
```

There are more typos like those, it looks like the content need to be proof-read before shipping.